lib/resources/datav2/rest_v2.ts

```diff
@@ -483,7 +483,7 @@ export async function getCryptoBars(
   symbols: string[],
   options: GetCryptoBarsParams,
   config: any
-): Promise<Map<string, any[]>> {
```

Comment:
Could you please change the rest of the `any` in the function as well?

Reply:
I haven't tested the result from other functions yet, but is this what you mean?

Reply:
Not exactly, I ment to change the `any` in the `getCryptoBars()` function to the proper type as well. (L494, L496)

Reply:
My assumption is that the internal types from the REST API is not converted to the return type before the for loop, is it safe to assume so?

Reply:
@noramehesz I reverted the last commit and added types inside function body.